app/controllers/plans_controller.rb

```diff
@@ -157,8 +196,11 @@ def update
     @plan = Plan.find(params[:id])
     authorize @plan
 
+    attrs = plan_params
+    attrs['visibility'] = Rails.application.config.default_plan_visibility if plan_params['visibility'].blank?
```

Comment:
Yes, a rake task would work here. We'll have to be sure to record it as a necessary step in the release notes though!